src-plugins/medVtkView/medVtkView.cpp

```diff
@@ -47,6 +47,7 @@ PURPOSE.
 #include <medParameterPoolManager.h>
 #include <medSettingsManager.h>
 #include <medClutEditorToolBox/medClutEditorToolBox.h>
+#include <medGlobal.h>
```

Comment:
That's a good remark, and I expected it, thanks!
I also thought about this when updating the branch.
But I am scared that medUtilities becomes the place where you put the functions that don't belong anywhere.
Might be better somewhere in medCore/database ?